app/controllers/podcasts_controller.rb

```diff
@@ -64,13 +64,13 @@ def create
 
   def sort_podcasts_by
     if params[:sort] == "Recent Activity"
-      "Recent Activity"
+      render html: '<span class="material-icons">timer</span>'.html_safe
```

Comment:
Something isn't quite working with the `material-icons-outlined` class as is, looking for solution